test/test_etl_api.py

```diff
@@ -0,0 +1,142 @@
+""" Unit test for etl API"""
+import time
+import unittest
+import requests
+import json
+import os
+import subprocess
+import shlex
+try:
+    from nameko.standalone.rpc import ClusterRpcProxy
+except ImportError:
+    raise ImportError("This unittest requires Nameko to be installed. You can install it with `pip install nameko`")
+
+# note: we should really make this part of a utils file in tests :-/
```

Comment:
This is even more generally useful. You could probably package this in its own repo. I could see using it for testing other projects.